verticapy/tests_new/core/vdataframe/test_plotting_animated.py

```diff
@@ -0,0 +1,91 @@
+"""
+Copyright  (c)  2018-2024 Open Text  or  one  of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+import pytest
+
+from IPython.display import HTML
+
+import verticapy as vp
+
+vp.set_option("plotting_lib", "plotly")
+print(vp.get_option("plotting_lib"))
+
+
+class TestPlottingAnimated:
+    """
+    test class for PlottingAnimated functions test
+    """
+
```

Comment:
I guess it is ok for now that we are only checking whether the output is HTML.

But I think we could check the content of HTML and find some salient features of our plot there as well.

Reply:
Okay